### ISSUE TITLE
Working on a similar hack for yt-1200 device

### ISSUE THREAD

**Issue by jkoenig72** [featherbear/zifon-yt1000-wifi-mcu#7]
Hi,

like your work. was able to replicate it for my yt1000, I also have a yt-1200 and try to apply same hack. 

I added a few pics below. Sadly board is different. But still using same radio modul. I can not identify the TX / RX pins, but I was thinking why not get those directly from the radio modul. But thats not working. Also I have issues identifiing the sequences for up/down/right/left. by listening to those pins, however I receive some hex data.

In your yt1000  hack you have attached th tx / rx pins differently, may I ask if you have tried also to attched them to the radio modul? As this seems obvious to me, And did that not work for you as well? I have put my yt1000 in action and would not like to dismantel it again... ;-)

Any idea what I can try? How to find the right tx/rx pins?

![yt1200_1](https://github.com/featherbear/zifon-yt1000-wifi-acu/assets/9740586/03130c6a-9dc7-4ced-a771-57e2e7fcbc52)
![yt1200_2](https://github.com/featherbear/zifon-yt1000-wifi-acu/assets/9740586/a37fabf7-8946-4b59-b518-602083e5457e)
![yt1200_3](https://github.com/featherbear/zifon-yt1000-wifi-acu/assets/9740586/44e52529-44d5-4a5c-977d-fc1b92134469)

**Comment by bojanv55** [comment]
I just did similar thing on the YT-1000. Using FT232RL FTDI USB 3.3V TTL Serial. I can read the data on the PC, and I can see it receives "START\r\n" and receives that sequences "60cc6060cc6060cc60..." when I use the remote, but when I try to send this data from the PC it seems it is not working. Not sure what could be the problem though...

**Comment by bojanv55** [comment]
**UPDATE**

I just tried with my Arduino UNO

```
#include <Arduino.h>

void setup() {
  Serial.begin(9600);
}

void loop() {
    Serial.write("\x60\xcc\x60");
}
```

and it works! so it seems that it is some problem with voltages used on USB-TTL I guess. Arduino probably works differently on TX pin.

**Comment by jkoenig72** [comment]
So - you can cofirm that directly soldering those pins to the remote works on the YT-1000?

Can you confirm?

If this is the case it should work on the YT-1200 as well. Then I must do something wrong.

**Comment by bojanv55** [comment]
yes. did you try arduino directly? I think the voltages for sending on USB-TTL are probably not what YT-1000 expects.

**Comment by jkoenig72** [comment]
I was setting it to 3.3v, but I will try over the weekend and let you know.

Thx!

**Comment by bojanv55** [comment]
Just to comment, that I tried also connecting uart5 of orangepi zero 2, and it also works OK.
